data/holidays.xml.in

```diff
@@ -368,6 +368,20 @@
     <date _name="7 of Hanuka" value="> hanuka(y)" offset="6"/>
     <date _name="8 of Hanuka" value="> hanuka(y)" offset="7"/>
   </country>
+  <country name="Italy">
```

Comment:
This attribute needs to be `_name` so that the country is marked for translation.

Reply:
Thank you. I modify it and send it again.

Reply:
My mistake. It didn't need the underscore in the _master_ branch.  I've fixed it.